src/Components/FlexBox/FlexBoxChild.tsx

```diff
@@ -48,6 +50,7 @@ export const FlexBoxChild = forwardRef<FlexBoxChildRef, FlexBoxChildProps>(
         style={flexBoxChildStyle}
         data-testid={testID}
         className={flexBoxChildClass}
+        onClick={onClick}
```

Comment:
`onClick` can be optional, right? what happens if `onClick` doesn't exist? do we need a no-op function in its place?